src/DssSpell.t.sol

```diff
@@ -58,7 +74,7 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(dai.balanceOf(WALLET2) - prev2, amt2);
     }
 
-    function testCollateralIntegrations() public { // make public to use
+    function testCollateralIntegrations() private { // make public to use
```

Comment:
I think specific tests content can be cleanup and/or replaced by templates introduced in previous spells eventually.

Reply:
`checkWards` tests could also be disabled as there is no new chainlog entry, to speed up testing.

Reply:
`checkAuth`

Reply:
done